addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -25,7 +25,7 @@
 
     --KanbanRecord__dropdown-gap: #{$border-width};
 
-    --KanbanColumn__highlight-background: #{rgba($o-action, 0.05)};
+    --KanbanColumn__highlight-background: #{tint-color($o-action, 90%)};;
```

Comment:
@edi-odoo , I just realised that this fails in dark-mode (rgba() fails too, but still).
Could you find a workaround?

Reply:
Fixed in enterprise